packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -108,7 +108,10 @@ describe("r2", () => {
 			  -c, --config                    Path to .toml configuration file  [string]
 			  -e, --env                       Environment to use for operations and .env files  [string]
 			  -h, --help                      Show help  [boolean]
-			  -v, --version                   Show version number  [boolean]"
+			  -v, --version                   Show version number  [boolean]
+
+			Options:
+			  -J, --jurisdiction  The jurisdiction where the new bucket will be created  [string]"
```

Comment:
We don't generally use capitalised flags in Wrangler. For consistency, would it be possible to use `-j`?

Reply:
-j seems to be in use already for --experimental-json-config. Would it be better to use -n?